website/content/docs/connect/gateways/api-gateway/configuration/routes.mdx

```diff
@@ -212,17 +151,29 @@ The following table describes the parameters for `path`:
 | `replacePrefixMatch` | Specifies a value that replaces the path prefix for incoming HTTP requests. The operation only affects the path prefix. The rest of the path is unchanged. | String | Required |
 | `type` | Specifies the type of replacement to use for the URL path. You can specify the following values: <ul><li>`ReplacePrefixMatch`: Replaces the matched prefix of the URL path (default). </li></ul> | String | Optional |
 
-### rules.matches
+### `rules.filters.extensionRef`
+
+Map that attaches an extension reference to an HTTP route when `rules.filters.type` is set to `ExtensionRef`. You can create extensions that enable different behaviors, such as defining JWT verification for requests to the route, and attach them to the route.
+
+You can configure the following parameters:
+
+| Parameter | Description | Data type | Default |
+| ---       | ---         | ---       | ---     |
+| `group`   | ?           | String    | None    |
```

Comment:
```suggestion
| `group`   | Specifies the resource group.           | String    | None    |
```
This is kind of a generic description. Perhaps the target audience would already understand what values to provide.